src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -1408,5 +1397,64 @@ public void BadKeepAlive4()
                 await serverData.Verify(connections: 2, completed: 2).ConfigureAwait(true);
             }
         }
+
+        [WorkItem(406649, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=406649")]
+        [Fact]
+        public void MissingCompilerAssembly()
```

Comment:
Added `MissingCompilerAssembly_CompilerServerHost` test, although that test is currently blocked by #19213.